libraries/lib-numeric-formats/formatters/BeatsNumericConverterFormatter.h

```diff
@@ -0,0 +1,21 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/**********************************************************************
+
+ Audacity: A Digital Audio Editor
+
+ @file ParsedNumericConverterFormatter.h
```

Comment:
wrong comment

Reply:
Really -- this is an unnecessary file!  When the .cpp accomplishes all its influence on program behavior by registrations, it can be headerless.  See src/menus or src/import for other examples.

Reply:
I need it for unit tests

Reply:
The same can be said of ParsedNumericConverterFormatter.h

Reply:
> I need it for unit tests

Good snswer

Reply:
Still, wrong comment